mappings/net/minecraft/world/biome/layer/SetBaseBiomesLayer.mapping

```diff
@@ -16,9 +16,12 @@ CLASS clw net/minecraft/world/biome/layer/BiomeGroupToBiomeLayer
 	FIELD o TAIGA_ID I
 	FIELD p SNOWY_TAIGA_ID I
 	FIELD q OLD_GROUP_1 [I
-	FIELD r GROUP_1 [I
-	FIELD s GROUP_2 [I
-	FIELD t GROUP_3 [I
-	FIELD u GROUP_4 [I
+	FIELD r DRY_CLIMATE_1 [I
+	FIELD s TEMPERATE_CLIMATE_2 [I
+	FIELD t COOL_CLIMATE_3 [I
+	FIELD u SNOWY_CLIMATE_4 [I
```

Comment:
Why are these numbered?

Reply:
They had numbers previously, don't know if they were important or not

Reply:
The numbers are the IDs. I can remove that if you want

Reply:
The point of the rename was to ditch the ambiguous numbering format

Reply:
Will remove the numbers. Thanks!

Reply:
Done